src/components/UpupUploader/DropZone.tsx

```diff
@@ -66,7 +66,7 @@ const DropZone: FC<Props> = ({
                         return multiple
                             ? [...prev, ...acceptedFiles]
                             : // only one file
-                              [acceptedFiles[0]]
+                              [acceptedFiles[0]].filter(Boolean)
                     })
```

Comment:
Could you explain this please @amjedidiah why did you add the .filter(Boolean) ?

Reply:
@MedAmine1212, updated to be more readable and safe.